node/flags/string_to_uint64.go

```diff
@@ -9,11 +9,11 @@ import (
 
 // StringToUint64Value is a flag type for string to uint64 values.
 type StringToUint64Value struct {
-	value map[string]uint64
+	value *map[string]uint64
 }
 
 // NewStringToUint64Value creates instance.
-func NewStringToUint64Value(p map[string]uint64) *StringToUint64Value {
+func NewStringToUint64Value(p *map[string]uint64) *StringToUint64Value {
```

Comment:
I feel like the naming for this type could be better? Maybe `StringUint64Map`?

Reply:
I think the author followed the convention in pflag library to name a type which implements the `Value` i/f this way (for example: [stringSliceValue](https://github.com/spf13/pflag/blob/2e9d26c8c37aae03e3f9d4e90b7116f5accb7cab/string_slice.go#L10)).